crates/sql-backend-handler/src/sql_migrations.rs

```diff
@@ -166,7 +167,13 @@ pub(crate) async fn upgrade_to_v1(pool: &DbConnection) -> std::result::Result<()
                 .col(ColumnDef::new(Users::PasswordHash).blob())
                 .col(ColumnDef::new(Users::TotpSecret).string_len(64))
                 .col(ColumnDef::new(Users::MfaType).string_len(64))
-                .col(ColumnDef::new(Users::Uuid).string_len(36).not_null()),
+                .col(ColumnDef::new(Users::Uuid).string_len(36).not_null())
```

Comment:
Oh, I was commenting on the wrong PR. Well then :)

Please don't change the v1, put all the changes in the v11 migration